components/org.wso2.carbon.identity.user.onboard.core.service/src/main/java/org/wso2/carbon/identity/user/onboard/core/service/password/ResetLinkGenerator.java

```diff
@@ -105,9 +105,9 @@ private UserRecoveryData generateNewConfirmationCode(User user, String notificat
         UserRecoveryDataStore userRecoveryDataStore = JDBCRecoveryDataStore.getInstance();
         userRecoveryDataStore.invalidate(user);
         String secretKey = Utils.generateSecretKey(notificationChannel, user.getTenantDomain(),
-                RecoveryScenarios.NOTIFICATION_BASED_PW_RECOVERY.name());
+                RecoveryScenarios.ADMIN_INVITE_SET_PASSWORD_OFFLINE.name());
         UserRecoveryData recoveryDataDO = new UserRecoveryData(user, secretKey,
-                RecoveryScenarios.NOTIFICATION_BASED_PW_RECOVERY, RecoverySteps.UPDATE_PASSWORD);
+                RecoveryScenarios.ADMIN_INVITE_SET_PASSWORD_OFFLINE, RecoverySteps.UPDATE_PASSWORD);
```

Comment:
Does this method get executed only for the offline invite flow?. If so thats fine.

Reply:
As we are changing the recovery scenario emitted in this flow, let's update our what has changed doc for IS 7.1 in case there was custom logic written by consuming the previous event.

Reply:
> Does this method get executed only for the offline invite flow?. If so thats fine.

Yeah this ResetLinkGenerator was introduced to support offline-user invite flow.
- https://github.com/wso2-extensions/identity-governance/pull/791